features/bootstrap/DatasetViewContext.php

```diff
@@ -332,26 +329,26 @@ public function iShouldSeeTabWithTable($arg1, TableNode $table)
     }
 
     /**
-     * @Then I sould not see :arg1 tab with table
+     * @Then I should not see :arg1 tab with table
      */
-    public function iSouldNotSeeTabWithTable($arg1, TableNode $table)
+    public function iShouldNotSeeTabWithTable($arg1, TableNode $table)
```

Comment:
You should check if any Behat tests were using the `iSouldNotSeeTabWithTable` function before you corrected its name. I think there are 2 tests in `dataset-view.feature` that used the `iSouldNotSeeTabWithTable` function. These tests need to be fixed if they are broken from your function name change.

Reply:
I think before I corrected it, `Scenario: Call To Actions - not logged in` has correct word `should`, and  only `Scenario: Files - Items per page` contains `sould`, so I changed it. 
And I have ran Behat test for `Scenario: Call To Actions - not logged in`, it passes too.

Reply:
But the rest of the `iSouldNotSeeTabWithTable` was used in the Samples table scenarios. I have updated them and Behat tests were passed.